### ISSUE TITLE
Why my trained results not so good?

### ISSUE THREAD

**Issue by Baiy97** [AndresPMD/StacMR#1]
Thanks for your codes. But when I trained the model with all hyperparameters unchanged, I got worse result compared to it in your paper, especially in image-to-text retrieval, shown as below.
![image](https://user-images.githubusercontent.com/26376415/105955881-ee64e680-60b1-11eb-97f6-d1a2de517b76.png)

The dataset I used for training is: Flickr30K (31784 images), TextCaps train set (21953 images) and CTC-caption validation set (5683 images).
As the visual features and OCR features of Flick30K and TextCaps are not provided, I used Faster-RCNN from [https://github.com/MILVLG/bottom-up-attention.pytorch](url) to extract visual features and used MaskTextSpotter from [https://github.com/MhLiao/MaskTextSpotter](url) to extract text token and embedded them with FastText.

I wonder if it is because I used different models from yours to extract visual features and OCR features, or there is something I have overlooked?

**Comment by AndresPMD** [comment]
Hello @Baiy97,

On my experience, the effect of different OCRs is significant. You can give it a shot with the visual and OCR features that I uploaded in the Readme file. 
Also, you can try using different val splits on the CTC (explicit and full) to evaluate the best performing set of models and eval them accordingly.
